a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -277,6 +277,8 @@ def get_delete_load_balancer_flow(self, lb, deleteCompute, cascade):
         delete_LB_flow.add(a10_database_tasks.SetThunderUpdatedAt(
             name=a10constants.SET_THUNDER_UPDATE_AT,
             requires=a10constants.VTHUNDER))
+        delete_LB_flow.add(network_tasks.DeallocateVIP(
```

Comment:
maybe we can use a10_network_tasks.DeallocateVIP()
it is the same, but seems we call ourown implementation in other places.